### DISCUSSION TITLE
Same status bar color as window

### DISCUSSION THREAD

**Discussion by tembbo** [catppuccin/tmux#451] [Help] [answered]
<img width="853" alt="Screenshot 2024-12-28 at 12 11 00 PM" src="https://github.com/user-attachments/assets/90b1b60b-67cb-4457-a612-682de0ca1461" />

How can I set the status bar color to match the window's background? I've tried multiple colors, but they either work for one type of window (e.g. terminal background but not neovim) or don't work at all.

**Comment by kjnsn** [comment, marked as answer]
Try `set -g status-style "bg=terminal"`. Make sure that's after the line in your config that loads catppuccin

**Reply by tembbo** [reply]
Thank you so much @kjnsn!
